controllers/admin/AdminPreferencesController.php

```diff
@@ -0,0 +1,253 @@
+<?php
```

Comment:
Seems like this file was removed from the branch, and you're re-adding it You can remove it from your branch as well